### PR TITLE
Add test coverage for dacpac wizard import flow

### PR DESCRIPTION
**Pull request by Benjin**
Also removes redundant variable definitions in dacfx wizard page classes

### CODE REVIEW

File: extensions/dacpac/src/test/configPages.test.ts [hunk 1]
```diff
@@ -74,4 +83,42 @@ describe('Dacfx Wizard Pages', function (): void {
...
+		wizard.setPages();
+
+		sinon.stub(azdata.connection, 'getConnections').resolves([mockConnectionProfile]);
+		sinon.stub(azdata.connection, 'listDatabases').resolves(['fakeDatabaseName']);
+		sinon.stub(vscode.window, 'showOpenDialog').resolves([vscode.Uri.file(dacpacPath)]);
+
+		let importConfigPage = new TestImportConfigPage(wizard, wizard.pages.get(PageName.importConfig).wizardPage, wizard.model, testContext.viewContext.view);
+		await importConfigPage.start();
+
+		//let result = await importConfigPage.onPageEnter();
```
Comment (kisantia) [inline extensions/dacpac/src/test/configPages.test.ts:100]: remove commented code

File: extensions/dacpac/src/wizard/pages/importConfigPage.ts [hunk 2]
```diff
@@ -54,30 +49,7 @@ export class ImportConfigPage extends DacFxConfigPage {
...
-				return;
-			}
-
-			let fileUri = fileUris[0];
-			this.fileTextBox.value = fileUri.fsPath;
-			this.model.filePath = fileUri.fsPath;
-			this.model.database = generateDatabaseName(this.model.filePath);
-			this.databaseTextBox.value = this.model.database;
-		});
+		this.fileButton.onDidClick(async (click) => { this.selectionPromise = this.handleFileSelection(); });
```
Comment (kisantia) [inline extensions/dacpac/src/wizard/pages/importConfigPage.ts:52]: does this need an await?

File: extensions/dacpac/src/test/configPages.test.ts [hunk 3]
```diff
@@ -74,4 +83,42 @@ describe('Dacfx Wizard Pages', function (): void {
...
+		//should(result).equal(true, 'onPageEnter() should successfullly load connection profiles');
+
+		testContext.viewContext.fileButtonOnClick.fire(undefined);
+		await importConfigPage.selectionPromise;
+		should(importConfigPage.Model.filePath).equal(dacpacPath);
+		should(importConfigPage.Model.database).equal('myDatabase');
+	});
 });
+
+const mockConnectionProfile: azdata.connection.ConnectionProfile = {
```
Comment (udeeshagautam) [inline extensions/dacpac/src/test/configPages.test.ts:110]: should we put this in a test util so other tests can use it too...

File: extensions/dacpac/src/test/testContext.ts [hunk 4]
```diff
@@ -102,7 +104,13 @@ export function createViewContext(): ViewTestContext {
 
 	let buttonBuilder: azdata.ComponentBuilder<azdata.ButtonComponent> = {
 		component: () => button,
-		withProperties: () => buttonBuilder,
+		withProperties: (properties) => {
```
Comment (udeeshagautam) [inline extensions/dacpac/src/test/testContext.ts:107]: you didnt need to change button builder to function? its great if this alone worked - ignore the comment if so. Just a caution that I noticed (for radio button) that was re-updating the same builder every time. So say it had simple click for a non '...' button. Then next time weget a buttin with '...' and change the onDidclick - it was still the same button object that changed so OnDidclick was updated for both buttons.

**Pull request closed without merging**